### PR TITLE
Solved Run length encoding exercise

### PR DESCRIPTION
**Pull request by iuliagatea**

### CODE REVIEW

File: Ruby/run-length-encoding/run_length_encoding.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  end
+
+  def initialize(data_string)
+    @data_string = data_string
+  end
+
+  def encode
+    string_chars = data_string.chars
+    string_chars.chunk(&:itself).map do |letter, array_of_letters|
+      "#{array_of_letters.count if array_of_letters.count > 1}#{letter}"
```
Comment (paucio) [inline Ruby/run-length-encoding/run_length_encoding.rb]: This line is kinda hard to follow.
Can you extract this condition into a private method?

File: Ruby/run-length-encoding/run_length_encoding.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    end.join('')
+  end
+
+  def decode
+    data_string.scan(/(\d*)(\D)/).map do |count, letter|
```
Comment (asarbu) [inline Ruby/run-length-encoding/run_length_encoding.rb]: 👍 another solution would have been using `gsub`

File: Ruby/run-length-encoding/run_length_encoding.rb (hunk 1 above)
Comment (iuliagatea) [reply Ruby/run-length-encoding/run_length_encoding.rb]: done
